docs/content/usage/configuration/credentials.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Credentials"
+date: 2020-06-09T12:20:08+03:00
+draft: false
+description: "Configure credentials that the Infection Monkey will use for propagation."
+---
+
+On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
```

Comment:
I think these pages can be removed. They explain less then configuration descriptions, so I don't see how they would help the user